conf/messages

```diff
@@ -97,7 +97,7 @@ error.no_entry_sa-postcode                                   = Enter the postcod
 
 ## Overseas Tax Identifier Page
 tax-identifier.title                                         = Does the business have an overseas tax identifier?
-tax-identifier.line_1                                        = We may use this number to help us identify your business. The tax identifier could be a VAT registration number, Employee Identification Number (EIN) or any other identifier we could use to verify your business details.
+tax-identifier.hint                                          = This may help us identify the business. It could be a Vat registration number, Employee Idenfication Number (EIN) or any other identifier we could use to verify the business details.
```

Comment:
please check the original ticket, there is a small change Vat -> VAT that needs to be done, in test below as well

Reply:
Just a small comment regarding the typo mistake I found in the ticket when doing ME is >>> "Idenfication"

Reply:
thanks @refaque-ahmad